### PR TITLE
Upgrade to SCS starters 2.1.2.RELEASE

### PR DESCRIPTION
**Pull request by habuma**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ initializr:
         - versionRange: "[1.5.0.RELEASE,2.0.0.M1)"
           version: 1.2.51
         - versionRange: "[2.0.0.M1,2.1.0.M1)"
-          version: 1.2.71  
+          version: 1.2.71
```
Comment (snicoll) [inline src/main/resources/application.yml]: That trailing whitespace seems unrelated. Can you please fix it?

Comment (habuma) [reply src/main/resources/application.yml]: Sure...my editor automatically removes trailing whitespace at the end of lines, which is why that is there. But I can probably put it back.

Comment (habuma) [reply src/main/resources/application.yml]: Oh geez...there were 2 spaces and I only put back 1.

Comment (habuma) [reply src/main/resources/application.yml]: Okay...fixed now.

Comment (habuma) [reply src/main/resources/application.yml]: @snicoll Is the change I've provided good for you? Anxious to get this merged to address a customer need.

**Comment by snicoll** [comment]
Sorry, afk. I’ll get to it in 30 min or so.
